Gopkg.toml

```diff
@@ -0,0 +1,79 @@
+# Force dep to vendor the code generators, which aren't imported just used at dev time.
```

Comment:
Does this branch predate the transition to Go modules?

Reply:
Yeah it does.

Reply:
Removed, thank you 🙏